packages/dataviews/CHANGELOG.automattic.md

```diff
@@ -16,6 +16,7 @@
 - Add `email` type to the fields of the form.
 - Add `className` prop to the `DataViews.Layout` component to allow customizing the layout styles.
 - Clean up --wp-components-color-* variables.
+- Add `help` prop support for `SelectControl` used in the `select` DataForm control, via the DataForm field `description` prop.
```

Comment:
The convention is to put the newest addition at the top :) (I forgot where we stated this convention)